src/Property/CSSNamespace.php

```diff
@@ -101,12 +96,12 @@ public function atRuleName(): string
     }
 
     /**
-     * @return array<int, string>
+     * @return array{0: CSSString|URL|string, 1?: CSSString|URL}
      */
     public function atRuleArgs(): array
     {
         $result = [$this->url];
-        if ($this->prefix) {
+        if ($this->prefix !== '') {
```

Comment:
This needs to also check it isn't `null`.  (I suspect this method isn't being fully tested.)

Reply:
Correct - this method is neither tested nor used.